docs/data-collection/intro.md

```diff
@@ -75,45 +80,95 @@ The above graph can be broken down as follows:
+| <img src="../../assets/images/ET_connections.jpg" alt="ET arm and connections" width="250px" /> | ![pctower](../assets/images/pctower.png) | ![infrared-mirror](../assets/images/infrared-mirror.png) |
+|:---:|:---:|:---:|
+| The ET is composed of three major elements: the ET arm (left), which holds the lens and camera and an infrared lamp, the *Host Computer* (middle), and the infrared mirror (right) {: colspan=3} | | |
 
-    ![gaz-analyser-front](../assets/images/gaz-analyser-front.jpg "Gas Analyzer front")
+* The ET arm is an articulated support for the infrared camera and lens in one end and with an infrared lamp at the other end.
+    The ET arm is installed inside the Scanning Room, inside the scanner's bore, standing on a transparent plexiglass panel tailored to the scanner's bed rail.
+    Two lengthy cables (black for power and orange for signal) are passed through the access panel and connect the ET arm to the ET computer.
+* The ET computer is a standard PC with customized software for the control of the ET.
+* The infrared mirror is critical to be able to record the **right eye** of the participant, and is mounted covering the standard mirror of the coil.
 
-    It is critical to familiarize with the [GA's manual](../assets/files/GA_manual.pdf) to learn about its correct utilization.
+## Scanning protocols
 
-    !!! danger "Make sure you understand the switching on and off procedures described in these SOPs"
+The study will collect data with two different scanning protocols.
+Most of what is described in the present SOPs addresses the *Reliability Imaging Protocol* that will be acquired on the BH07060 scanner.
 
-    The input to the GA must be connected through a DM-060-24 desiccant chamber and a MLA0110 flow valve and a MLA0343 drying tube:
+| **Reliability Imaging Protocol (36 sessions $\times$ one scanner)** | mm:ss | **Standard Imaging Protocol (12 sessions $\times$ three scanners)** | mm:ss |
+|---|---|---|---|
+| Head scout | 00:14 | Head scout | 00:14 |
+| FoV Manual positioning | 01:00 | FoV Manual positioning | 01:00 |
+| T1w (anatomical reference) | 5:41 | T1w (anatomical reference) | 5:41  |
+| DWI (dMRI, structural connectivity) | 33:52 | EPI BOLD ($B_\text{0}$ field mapping) | 00:30 |
+| DWI ($B_\text{0}$ field mapping) | 00:54 | BOLD (single-echo RSfMRI, eyes open) | 20:09 |
+| GRE ($B_\text{0}$ field mapping) | 02:38 | GRE ($B_\text{0}$ field mapping) | 03:08 |
+| EPI BOLD ($B_\text{0}$ field mapping) | 00:27 | BOLD (multi-echo RSfMRI, eyes open, **only BH07060**) | 10:09 |
+| BOLD (positive-control task fMRI) | 03:07 | DWI (dMRI, structural connectivity) | 27:31 |
+| BOLD (multi-echo RSfMRI, naturalistic movie) | 20:09 | DWI ($B_\text{0}$ field mapping) | 01:10 |
+| BOLD (breath-holding task fMRI) | 06:00  | T2w (anatomical reference) | 08:24 |
+| T2w (anatomical reference) | 05:10  | | |
+| **Total Acquisition Time** | 79:12 | **Total Acquisition Time** | 77:56 |
```

Comment:
Would it be relevant to have a vertical line to distinguish between the two protocols ?

Reply:
Hahahaha, it would be great. Can you investigate and suggest changes to get the markdown to draw it?

Reply:
It appears the style is inheritent from Material for mkdocs and we'll have to provide additional options in the css file to show vertical lines. I tried it and it's ugly (it shows all lines) - don't think we can display specific lines that way.

I'd say it's not a priority :)

Reply:
Yeah, we probably want to split this table in some future. After all, the generalization protocol is barely talked about in the SOPs.